brands/shop/convenience.json

```diff
@@ -373,6 +373,8 @@
     "count": 475,
     "tags": {
       "brand": "Costcutter",
+      "brand:wikidata": "Q5175072",
+      "brand:wikipedia": "en:Costcutter",
```

Comment:
This also exists in the supermarket section: https://github.com/osmlab/name-suggestion-index/blob/5e6f2a4e6441f3fc74d2174d91e2239e491e8dfc/brands/shop/supermarket.json#L776-L785

Do you know whether this store is considered rather as a supermarket or a convenience store?
Judging only from the amount of mapped places, I would say convenience is the right one...

Reply:
According to Wikipedia "It encompasses both supermarkets and convenience shops" because of mergers and whatnot. I guess they run a bunch of former SPAR locations in the United Kingdom etc. So both entries are correct.

Reply:
I will add nomatches after merging PR.

https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md#nomatch

Reply:
Most of the Costcutters I've seen mapped as supermarkets are what I would call convenience stores, though some of them are large enough to be small supermarkets so both tags should be kept . Wikipedia says they have some stores in Poland as well which might be supermarkets but none of those have been mapped so far.